new_dawn_server/pusher/notification_service.py

```diff
@@ -0,0 +1,27 @@
+from pusher_push_notifications import PushNotifications
+from django.conf import settings
+
+
+beams_client = PushNotifications(
+            instance_id=settings.BEAMS_INSTANCE_ID,
+            secret_key=settings.BEAMS_SECRET_KEY,
+        )
+
+
+class NotificationService():
+
```

Comment:
Let's create another function called _get_instance_id_and_secret_key(). And use this to retrieve instance id and secret key for the client.

Then in the test, lets patch this function to return dummy instance id and secret key